Week1/Nadege_Awah/solution_to_palindrome_exercise.js

```diff
@@ -0,0 +1,15 @@
+//defines function reverseString and gives it a parameter str
+function reverseString(str) { 
+  var palindrome = "str"
+  /*
+  str is converted to array, array is reversed and converted back to string and checked if the result is equal to str.
+  */
+  if((str.split("").reverse("").join("")) == str) {
```

Comment:
Well the solution works. 
Can you think of a way to take out this `if` `else` statement?
```suggestion
  if((str.split("").reverse("").join("")) == str) {
```

Reply:
Ok. Thanks for the feedback.